LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Jean-Philippe Doyle, Copyright (c) 2015
+Copyright (c) 2016
```

Comment:
Didn't mean to remove it, was running `ember init` and thought the only change was the year.